.changeset/twelve-bears-fail.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/core-app-api': patch
+'@backstage/core-plugin-api': patch
```

Comment:
Shouldn't this be `minor` as it's now been removed?

Reply:
👍 , added a separate changeset for the removal